internal/controller/chiaca/controller.go

```diff
@@ -81,89 +78,38 @@ func (r *ChiaCAReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 		return ctrl.Result{}, fmt.Errorf("ChiaCAReconciler ChiaCA=%s encountered error querying for existing CA Secret: %v", req.NamespacedName, err)
 	}
 
-	// If CA exists, just set ChiaCA status to ready and return
-	if caExists {
-		if !ca.Status.Ready {
-			r.Recorder.Event(&ca, corev1.EventTypeNormal, "Created",
-				fmt.Sprintf("Successfully created CA Secret in %s/%s", ca.Namespace, ca.Name))
-
-			ca.Status.Ready = true
-			err = r.Status().Update(ctx, &ca)
-			if err != nil {
-				if strings.Contains(err.Error(), kube.ObjectModifiedTryAgainError) {
-					return ctrl.Result{RequeueAfter: 1 * time.Second}, nil
-				}
-				metrics.OperatorErrors.Add(1.0)
-				log.Error(err, "encountered error updating ChiaCA status")
-				return ctrl.Result{}, err
-			}
+	// If CA Secret doesn't exist, generate a CA and create one
+	if !caExists {
+		chiaCACrt, chiaCAKey := tls.GetChiaCACertAndKey()
+		privateCACrt, privateCAKey, err := tls.GenerateNewCA("")
```

Comment:
`GenerateNewCA` returns the cert-key pair in byte slices and an error, but also outputs them to a file, which is less than ideal here. There should ideally be a way in go-chia-libs to do this without writing to a file, and then change this to use that.